src/exodus/wallet.h

```diff
@@ -0,0 +1,71 @@
+#ifndef EXODUS_WALLET_H
+#define EXODUS_WALLET_H
+
+#include <string>
+
+#include "sigma.h"
+#include "walletdb.h"
+#include <wallet/wallet.h>
```

Comment:
This one should be relative path due to it will broken if build system did not add `src` to includes path.